spec/features/ctc/complete_intake_spec.rb

```diff
@@ -33,13 +33,15 @@ def strip_html_tags(text)
     expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.filed2020.title'))
     click_on I18n.t('general.negative')
     expect(page).to have_selector("h1", text: I18n.t('views.ctc.questions.filed2019.title'))
-    choose I18n.t('views.ctc.questions.filed2019.did_not_file')
+    choose I18n.t('views.ctc.questions.filed2019.filed_full')
```

Comment:
there's a separate "client who has filed in 2019" scenario in this file that would be a better place to put this, so we have one flow for a non-filer and another test that does the AGI for the primary and spouse

Reply:
done!